src/utilities/aspect-ratio/aspect-ratio.scss

```diff
@@ -0,0 +1,21 @@
+@use 'sass:meta';
+@use '../../compiled/tokens/scss/aspect-ratio';
+@use '../../mixins/media-query';
+
+/// Saving a map of aspect ratio values makes it easier to loop through.
+/// @type Map
+$aspect-ratio-map: meta.module-variables('aspect-ratio');
+
+.u-aspect-ratio {
+  @each $name, $ratio in $aspect-ratio-map {
+    &-#{$name} {
+      @include media-query.breakpoint-classes {
+        --aspect-ratio: #{$ratio};
+        aspect-ratio: var(--aspect-ratio);
+        block-size: 100%;
+        inline-size: 100%;
+        object-fit: cover;
```

Comment:
Wasn't there a bug with Safari video embeds and this property?

Reply:
Oh, good catch. I didn't _play_ the video. 🤦🏽

Reply:
Hmm...the `video` element works fine. It's the YouTube `<iframe>` embed that has the issue. 

If I add `object-position: left`, it seems to fix the YouTube `<iframe>` embedded video but then all of the rest of the examples (e.g. image or picture) are also get repositioned `left` which is not what we want.

Reply:
> If I add `object-position: left`, it seems to fix the YouTube `<iframe>` embedded video but then all of the rest of the examples (e.g. image or picture) also get repositioned `left` which is not what we want.

Or, if I _remove_ `object-fit: cover`, then the Safari issue goes away as well. But then the image assets get squished to fit within their aspect ratio space. 🤔

Reply:
I suppose, if [I did switch to an object or component](https://github.com/cloudfour/cloudfour.com-patterns/pull/2053#discussion_r978114186), I could have a "YouTube" modifier that unsets `object-fit: cover` maybe?

Reply:
Or you could move that to a separate utility.

Or you could qualify that property to only `img` elements?

Reply:
I found a fix for the Safari bug. Thanks for the idea, @tylersticka! https://github.com/cloudfour/cloudfour.com-patterns/pull/2053/commits/21cac28d0886d7e371b13f81b2547e324800b698